tempest/api/workloadmgr/barbican/test_barbican.py

```diff
@@ -3346,3 +3346,83 @@ def test_13_cleanup(self):
             self.delete_secret(secret_uuid)
         except Exception as e:
             LOG.error(f"Exception in test_13_cleanup: {e}")
+
+    #OS-2018 -
+    #Create secret with empty payload.
+    #Create encrypted workload with above secret UUID.
+    #Result - Workload creation should fail as secert UUID should map to payload
+    @decorators.attr(type='workloadmgr_cli')
+    def test_14_barbican(self):
+        try:
+
+            test_var = "tempest.api.workloadmgr.barbican.test_"
+            tests = [[test_var + "create_secret_with_empty_payload", 0]]
+            reporting.add_test_script(tests[0][0])
+
+            #create key pair...
+            self.kp = self.create_key_pair(tvaultconf.key_pair_name)
+
+            #create vm...
+            self.vm_id = self.create_vm(key_pair=self.kp)
+
+            #create secret with empty payload.
+            empty_payload = "\"\""
+            secret_key_cmd = command_argument_string.openstack_create_secret + \
+                                  "-p " + str(empty_payload) + \
```

Comment:
-p needs to be part of command_argument_string only.

Reply:
Handled in command_argument_string.py